report/finished/chapters/modeling.tex

```diff
@@ -1,6 +1,206 @@
-\section*{Robotic arm}
-\subsection*{Forward kinematics}
-\subsection*{Inverse kinematics}
+
+%%%%%%%%%%%%%%%% arm kinematics %%%%%%%%%%%%%%
+
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=0.6\textwidth]{chapters/img/robot_arm.png}
+    \caption{Placeholder image for the robotic arm?.}
+    \label{fig:robotic_manipulator_general}
+\end{figure}
+
+
+\section*{Manipulator Kinematics} %Niklas
+Kinematics is what describes the motion of rigid bodies and points in space. 
+
+\subsection*{Backgroud}
+\subsection*{Rigid body transformation}
+In general any position can be described by translation along 3 axes and rotation along these same axes. These translations/rotations can be described by a matrix
+\begin{equation}
+    T = 
+    \begin{bmatrix}
+        R & d \\
+        \bf{0} & 1
+    \end{bmatrix}
+\end{equation}
+where \(R\) is a 3 x 3 rotation matrix and \(d\) a 3 x 1 translation matrix. This implies that any transformation could be characterized by six parameters, three for the translation and three for the rotation. \cite{spong}
+
+
+\subsection*{Denavit-Hartenberg Convention}
+A common approach for selecting the coordinate frames of reference for each joint in a robotic arm is to us the Denavit-Hartenberg convention. This allows the transformation matrix for each joint to be expressed as
```

Comment:
use

Reply:
would just remove is to use:
A common approach for selecting the coordinate frames of reference for each joint in a robotic arm is the Denavit-Hartenberg convention.

Reply:
I am doing what @edwced suggested and removing the "to use".